packages/editor/__tests__/specs/userEditPanel.tsx

```diff
@@ -110,27 +96,23 @@ describe('User Edit Panel', () => {
     const mocks = [userDocumentQuery, userRoleListDocumentQuery]
 
     const wrapper = mount(
-      <UIProvider styleRenderer={styleRenderer} rootElementID={'fskr'}>
-        <MockedProvider mocks={mocks} addTypename={true}>
-          <UserEditPanel id={'fakeId3'} />
-        </MockedProvider>
-      </UIProvider>
+      <MockedProvider mocks={mocks} addTypename={true}>
+        <UserEditPanel id={'fakeId3'} />
+      </MockedProvider>
     )
     await updateWrapper(wrapper, 100)
 
     const panel = wrapper.find('UserEditPanel')
     expect(panel).toMatchSnapshot()
   })
 
-  test('should allow user role to be added', async () => {
+  xtest('should allow user role to be added', async () => {
```

Comment:
'xtest' a typo?

Reply:
No. I need to fix those before the merge 😄

Reply:
With the `x` you can disable tests.

Reply:
Got it  :)